docs/release-notes/2.16.0.md

```diff
@@ -0,0 +1,138 @@
+
+* [PMM-7339](https://jira.percona.com/browse/PMM-7339): Security Threat Tool: Ability to execute Security checks individually and on-demand
+
+* [PMM-7451](https://jira.percona.com/browse/PMM-7451), [PMM-7337](https://jira.percona.com/browse/PMM-7337): Security Threat Tool: Ability to change intervals for Security checks on the Settings page
+
+* [PMM-7772](https://jira.percona.com/browse/PMM-7772), [PMM-7338](https://jira.percona.com/browse/PMM-7338): Security Threat Tool: Ability to change default execution interval per check
+
+* [PMM-7336](https://jira.percona.com/browse/PMM-7336): Security Threat Tool: Execute checks based on execution interval they belong to
+
+* [PMM-7335](https://jira.percona.com/browse/PMM-7335): Security Threat Tool: Ship security check files with predefined execution interval
+
+* [PMM-7748](https://jira.percona.com/browse/PMM-7748): Add an additional experimental menu for Dashboards on the left side panel
+
+* [PMM-7688](https://jira.percona.com/browse/PMM-7688): Unify UX and layout of all PMM specific pages like Settings, Add Instance etc,
+
+* [PMM-7687](https://jira.percona.com/browse/PMM-7687): Modify links in menus to ensure both menus are working as expected after dashboard URL change
+
+* [PMM-7705](https://jira.percona.com/browse/PMM-7705): Consistent support of Technical Preview Features
+
+* [PMM-7522](https://jira.percona.com/browse/PMM-7522), [PMM-7511](https://jira.percona.com/browse/PMM-7511): Integrated Alerting: Improve Notification Channels UX by  Pagination for the Notification list
+
+* [PMM-7521](https://jira.percona.com/browse/PMM-7521), [PMM-7510](https://jira.percona.com/browse/PMM-7510): Integrated Alerting: Improve Alert Rule Templates UX by  Pagination on Rule Templates list
+
+* [PMM-7652](https://jira.percona.com/browse/PMM-7652), [PMM-7674](https://jira.percona.com/browse/PMM-7674), [PMM-7503](https://jira.percona.com/browse/PMM-7503), [PMM-7486](https://jira.percona.com/browse/PMM-7486): DBaaS: While creating the DB cluster  see all and available resources in the K8s cluster, such as Disk, CPU & Memory
+
+* [PMM-7508](https://jira.percona.com/browse/PMM-7508), [PMM-7488](https://jira.percona.com/browse/PMM-7488): DBaaS: See predicted resource usage for selected DB Cluster configuration
+
+* [PMM-7364](https://jira.percona.com/browse/PMM-7364): DBaaS: Show warning before starting creating the cluster if there are not enough resources in the K8s cluster to create DB Cluster with requested configuration
+
+* [PMM-7580](https://jira.percona.com/browse/PMM-7580), [PMM-7359](https://jira.percona.com/browse/PMM-7359): DBaaS: Users can select the database version to use during DB Cluster creation
+
+## Improvements
+
+* [PMM-7506](https://jira.percona.com/browse/PMM-7506): Security Threat Tool: Exclude PXC internal user accounts from mysql_empty_password check
```

Comment:
Ummm...do we want to disclose this?  if it's a security threat to have an unset password how is it less of a threat to have blank passwords in DBaaS?  

Is there better way to say "Do as I say, not as I do"?  How about 'Exclude DBaaS created DB's from certain checks'

Reply:
This is not DBaaS-related, it's  PXC in general. To reduce false-positive STT alerts

Reply:
Then we should probably word it that way...this looks like we are giving ourselves a pass on a security check when it's the nature of how PXC works...so instead I'd recommend we say "Reduce False Positive's for PXC's internal user account"